packages/neo-one-server-plugin-wallet/src/bootstrap.js

```
@@ -0,0 +1,287 @@
+  if (networkName != null && typeof networkName === 'string') {
+    return networkName;
+  }
+
+  const { network } = await cli.getSession(networkConstants.PLUGIN);
+  if (network != null && typeof network === 'string') {
+    return network;
+  }
+
+  throw new Error(
+    'Bootstrap requires a network. Activate a network by ' +
+      'running `activate network <name>` or specify the network via ' +
+      '`--network <name>`',
+  );
+};
+
+async function getWallet(
+  walletName: string,
+  networkName: string,
+  cli: InteractiveCLI,
+  plugin: WalletPlugin,
+): Promise<Wallet> {
+  const wallet = await plugin.walletResourceType.getResource({
+    name: constants.makeWallet({
+      network: networkName,
+      name: walletName,
+    }),
+    client: cli.client,
+    options: {},
+  });
+
+  if (wallet == null) {
+    throw new Error(`Failed to find wallet, ${walletName}`);
+  }
+
+  return wallet;
+}
+
+async function createWallet(
+  walletName: string,
+  networkName: string,
+  cli: InteractiveCLI,
+  keystore: LocalKeyStore,
+): Promise<void> {
+  const walletResource = await cli.client.createResource({
+    plugin: constants.PLUGIN,
+    resourceType: constants.WALLET_RESOURCE_TYPE,
+    name: constants.makeWallet({ name: walletName, network: networkName }),
+    options: {
+      network: networkName,
+    },
+    cancel$: _of(),
+  });
+
+  const wallet = ((walletResource: $FlowFixMe): Wallet);
+  if (wallet == null) {
+    throw new Error(`Failed to find wallet, ${walletName}`);
+  }
+  if (wallet.wif == null) {
+    throw new Error(`Something went wrong, wif is null for ${walletName}`);
+  }
+
+  await keystore.addAccount({
+    network: networkName,
+    name: wallet.name,
+    privateKey: wifToPrivateKey(wallet.wif),
+  });
+}
+
+function getNumWallets(options): number {
+  const { wallets } = options;
+  if (wallets != null && typeof wallets === 'number') {
+    return wallets;
+  }
+  return DEFAULT_NUM_WALLETS;
```

Should throw an error if `wallets` is provided and it's not a number.